packages/angular/src/errorhandler.ts

```diff
@@ -40,7 +41,9 @@ class SentryErrorHandler implements AngularErrorHandler {
     const extractedError = this._extractError(error) || 'Handled unknown error';
 
     // Capture handled exception and send it to Sentry.
-    const eventId = runOutsideAngular(() => Sentry.captureException(extractedError));
+    const eventId = runOutsideAngular(() =>
+      getCurrentHub().captureException(extractedError, { data: { mechanism: { type: 'angular', handled: false } } }),
+    );
```

Comment:
Can we change the way of passing on the mechanism to using an eventprocessor on the scope? Similarly to how we do it in [remix here](https://github.com/getsentry/sentry-javascript/blob/195fe33df1e42383cc05a1a58b607d9b8728bf15/packages/remix/src/utils/instrumentServer.ts#L73-L85)? If this is possible, we'd prefer it method over adding it to the hint.